index.html

```diff
@@ -118,6 +118,16 @@ <h2>Join thousand of happy customers</h2>
 
 
     <!-- free trial: start -->
+        <section class="container">
+            <div class="row">
+                <div class="col-12">
+                    <h2>Try fror 7 days free</h2>
+                    <p>Each time a digital asset is purchased or sold, Sequoir donates a percentage of the fees back.</p>
+                    <form>
+                        FORM
+                    </form>
```

Comment:
Truksta buttono.

Reply:
As mastau kad bus kaip for dalis ten input galima padaryti submit...
Ka manone? 
nes per paskaitais formom irgi button nekure..

Reply:
Yra tiesos, kolkas manau paliekam taip.

Reply:
Ok dar gal Povilas tures kokiu ideju, ryt galesim Kestucio ar Rimanto pasiklausti..

Reply:
as manau, kad button nereik dar det, viskas cia ok, tik max ka padaryciau, tai nebent pakisciau h2+p po vienu extra divu ir taip paciai dorma pakisciau po divu dar del visa ka, bet cia galim ir ateiti padaryt, kai jau kontruosim pilnai.

Reply:
Pridejau extra divus.. Gera ideja..